infra/tf/README.md

```diff
@@ -7,4 +7,35 @@ Terraform is our tool of choice for automating our 'cloud infrastructure'. In pa
 ## Contents
 
 * Something about Truss Terraform Modules
-* How to layout/structure a Trraform Project
+* How to layout/structure a Terraform Project
+
+### Starting a module experiment
+
+...
+
+### Going public
+
+When you're ready to turn the prototype module into a published one, there's a few things you'll need to do if you haven't already:
+
+1. 🔒 Add your repo to the [list of repos](https://github.com/trussworks/truss-infra/blob/master/modules/github-truss-repository/main.tf) managed by our GitHub repo management module.
+2. 🔒 Add a subscription to your repo in our [#infra-feed](https://trussworks.slack.com/messages/C91SHMKFV/) channel:
+   * `/github subscribe trussworks/your-repo-here`
+   * `/github unsubscribe trussworks/your-repo-here statuses commits deployments public releases`
+3. Wire the repo up to the [terraform module registry](https://registry.terraform.io).
+
+### Publishing a release
+
+To publish a release from a terraform module that's already connected to the [terraform module registry](https://registry.terraform.io), all you need to do after your PR merges is:
+
+1. From the releases tab, click the "Draft a release" button.
```

Comment:
The draft a release step includes specifying a version, is the tag created as part of the release process? If not we should document how to do that. Also, any guidelines on how to increment the version we should follow probably could be included here.

Reply:
I add my thoughts on that based on how we did at at $oldjob & in npm.

Reply:
I can't find any recommendations from terraform so I think the standard pattern from npm and others is good.

Reply:
I found a better source to reference than npm here. :)